helm/cluster-api-provider-azure/templates/rbac.yaml

```diff
@@ -118,6 +118,7 @@ rules:
       - watch
   - apiGroups:
       - exp.cluster.x-k8s.io
+      - cluster.x-k8s.io
```

Comment:
It seems the `status` is missing from here https://github.com/giantswarm/cluster-api-provider-azure-app/blob/3953a5e9d7a3749b56b8540953c47f08060c0721/helm/cluster-api-provider-azure/templates/rbac.yaml#L96